documentation/tutorials/getting-started-with-ash-json-api.md

```diff
@@ -68,7 +80,8 @@ defmodule HelpdeskWeb.JsonApiRouter do
     # The api modules you want to serve
     domains: [Module.concat(["Helpdesk.Support"])],
     # optionally an open_api route
-    open_api: "/open_api"
+    open_api: "/open_api",
+    prefix: "/api/json"
```

Comment:
Igniter adds this extra line.

Reply:
I believe this is fixed in the latest version of `ash_json_api`

Reply:
I have removed this part. But FYI, I used the latest version of `ash_json_api` and it added this extra line.

Reply:
Perhaps it wasn't published, but the  source code definitely doesn't add it.